### PR TITLE
fix: show label in textField

### PR DESCRIPTION
**Pull request by FariasBlu**
## Description

This PR fix show label of InputSelect

Fixes # (issue)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Screenshots (if appropriate):

https://github.com/user-attachments/assets/e7802bb9-89f8-4cc3-9814-3a4a190ae8c9

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

**Review by vromeiro — APPROVED** [review]

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift [hunk 1]
```diff
@@ -132,6 +138,7 @@ extension OceanSwiftUI {
                         .font(Font(UIFont.baseRegular(size: Ocean.font.fontSizeXs)!))
                         .foregroundColor(Color(self.parameters.isDisabled ? Ocean.color.colorInterfaceLightDeep : Ocean.color.colorInterfaceDarkDeep))
                         .oceanSkeleton(with: self.parameters.showSkeleton)
+
```
Comment (acassiovilasboas) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:141]: Celseira tem umas 3 linhas aqui... 🔪 nelas

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift [hunk 2]
```diff
@@ -95,11 +95,17 @@ extension OceanSwiftUI {
         // MARK: View SwiftUI
 
         private var textFieldView: some View {
-            TextField(self.parameters.placeholder, text: self.$parameters.text, onEditingChanged: { edit in
-                self.focused = edit
+            SwiftUI.Button(action: {
+                self.openModal()
+            }, label: {
+                HStack(spacing: Ocean.size.spacingStackXxs) {
+                    OceanSwiftUI.Typography.description { label in
```
Comment (acassiovilasboas) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:102]: Se vc usar o Typography.heading4 ele ja é com fonte 16 e vc poderia remover a linha que define a fonte.

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift [hunk 3]
```diff
@@ -95,11 +95,17 @@ extension OceanSwiftUI {
...
+                        label.parameters.font = .baseRegular(size: Ocean.font.fontSizeXs)!
+                        label.parameters.text = self.parameters.text.isEmpty ? self.parameters.placeholder : self.parameters.text
+                        label.parameters.textColor = self.parameters.isDisabled ? Ocean.color.colorInterfaceLightDeep : self.parameters.text.isEmpty ? Ocean.color.colorInterfaceDarkUp : Ocean.color.colorInterfaceDarkDeep
```
Comment (acassiovilasboas) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:105]: Eu sou fã do if ternario mas quando a gente aninha mais uma condicao eu acho que fica mais dificil de ler, o que vc acha de levar essa logica para uma funcao getColor ou uma variavel computada?
```
let textColor: Color = {
    if self.parameters.isDisabled {
        return Ocean.color.colorInterfaceLightDeep
    } else if self.parameters.text.isEmpty {
        return Ocean.color.colorInterfaceDarkUp
    } else {
        return Ocean.color.colorInterfaceDarkDeep
    }
}()
````

```
private func getTextColor() -> Color {
    if self.parameters.isDisabled {
        return Ocean.color.colorInterfaceLightDeep
    } else if self.parameters.text.isEmpty {
        return Ocean.color.colorInterfaceDarkUp
    } else {
        return Ocean.color.colorInterfaceDarkDeep
    }
}
```

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift [hunk 4]
```diff
@@ -95,11 +95,17 @@ extension OceanSwiftUI {
...
 
         private var textFieldView: some View {
-            TextField(self.parameters.placeholder, text: self.$parameters.text, onEditingChanged: { edit in
-                self.focused = edit
+            SwiftUI.Button(action: {
+                self.openModal()
+            }, label: {
+                HStack(spacing: Ocean.size.spacingStackXxs) {
+                    OceanSwiftUI.Typography.description { label in
+                        label.parameters.font = .baseRegular(size: Ocean.font.fontSizeXs)!
```
Comment (acassiovilasboas) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:103]: A propriedade font do typography é nulable, nao precisa forçar com "!" aqui.
image

**Review by acassiovilasboas — APPROVED** [review]

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift (hunk 1 above)
Comment (FariasBlu) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:141]: Eeeeeita

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift (hunk 2 above)
Comment (FariasBlu) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift:102]: Feito

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift (hunk 3 above)
Comment (FariasBlu) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift]: Bora com a func

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift (hunk 4 above)
Comment (FariasBlu) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputSelectField.swift]: Feito

**Pull request merged**